services/update.py

```diff
@@ -116,7 +116,8 @@ def is_valid(self):
         sql = """SELECT id, status, addontype_id, guid FROM addons
                  WHERE guid = %(guid)s AND
                        inactive = 0 AND
-                       status != %(STATUS_DELETED)s
+                       status != %(STATUS_DELETED)s AND
+                       is_listed != 0
```

Comment:
Since `guid` is unique I'm assuming there is no database index to modify, right ?